src/components/backend-ai-storage-proxy-list.ts

```diff
@@ -418,6 +418,18 @@ export default class BackendAIStorageProxyList extends BackendAIPage {
     return;
   }
 
+  /**
+     * Determine if the storage has unmatched file system.
+     *
+     * @return {boolean}
+     */
+  unmatchedFileSystem() {
+    if (this.agentDetail.cpu_util_live == null ) return true;
```

Comment:
If there's VFS only, we cannot initialize property `agentDetail` since It initialized in a certain plugin. How about using some agent data received from the manager server?